frontend/PROPOSAL.md

```diff
@@ -0,0 +1,113 @@
+# Proposal for refactoring the project (frontend)
+
+**Index**
+
+1. [Folder Structure](#folder-structure)
+2. [React Component Conventions](#react-component-conventions)
+3. [CSS Conventions](#css-conventions)
+
+## Folder Structure
+
+```
+frontend
+└── src
+    ├── components (for all global components)
+    │    ├── Header
```

Comment:
can we please have the component and directory names in small case like here `header`instead of `Header` and similarly at other places?

Reply:
Hi @Kajol-Kumari. I feel its better to follow the PascalCase naming convention for the component files and its directories as these conventions are followed by the major companies. 

Also, I did not understand where you want the spaces to be replaced by `-`

Reply:
Hmm, I see. Incase the component name is too big, I was thinking of using `-` inplace of spaces

Reply:
I have not added space anywhere. 🤔  In case the component name is big, PacalCase will take care of it. 

For example, `HeaderComponentWithNavigation`. Of course, this is less likely to happen as most of the component names will comprise 2 words at the most. 

`-` we can use for SCSS files. For example, `header-component.modules.scss`

I have used `kebab-case` convention for SCSS files because we use a similar style for CSS properties. Example, `background-color`, `font-size`, etc. 

*Note* - camelCase can also be used for SCSS/CSS file names. `headerComponent.modules.scss`